README.md

```diff
@@ -1,11 +1,283 @@
+18. [Next Steps – Controlling Ads via your O2 Portal Account](#co2)
+
+
+## What is the O2 Mobile SDK? <a name="what"></a>
+The O2 Mobile SDK (OMSDK or SDK) is a native iOS SDK with the sole purpose for playing and monetizing videos from the Oath O2 video platform in your app. The OMSDK is written in Swift and is delivered as a framework. You can include this in your app projects either via CocoaPods or Carthage. Currently, Swift Package Manager not supported on iOS or tvOS.
+
+As part of playing videos, the OMSDK also handles video ads (pre-roll, mid-roll, and post-roll) and associated videos and ads playback and performance analytics. Analytics are focused on tracking what is played, how far it is played (e.g., deciles, quartiles), and details about the actual device or network. For more details on the analytics supported or to access the analytics data, you will work with the [Video Support Team](mailto:video.support@oath.com) to build reports that focus specifically on your app’s video and ads performance.
+
+The SDK includes a complete default video player controls UX (user experience), which includes a limited albeit robust set of customization options. The controls implementation is fully open source, and the SDK architecture allows for you to include your own fully customized controls UX, should you not be interested in the built-in default one.
+
+## Main SDK Features <a name="features"></a>
+* Playback of one or more individual videos or a single playlist of videos
+* Video playback of VOD (video on demand), 360°, and LIVE streaming video types
+* Supports either .mp4 or .m3u8 (HLS) formats
+* Video ads (VAST support of .mp4 format only)
+* Tap an ad to access ad URL (more information) via an in-app-browser
+* Full video and ads analytics
+* Default video player controls UX (full source code open sourced)
+* HLS support for multiple closed captioning (CC) and SAP audio languages
+* Mute/Unmute support
+* Automatic filtering of geo-restricted videos
+* Complete apps control of the frame where videos play
+* Native iOS Picture-in-picture support
+* Apple AirPlay support
+
+## Why would I use the O2 Mobile SDK? <a name="why"></a>
+The O2 Mobile SDK is used to natively play O2 videos. If you have a native app, you should use the SDK. The main reason as to why you’d want to use the SDK, is because you get all the ads and analytics for free. The ads are important for monetization. Analytics are important for tracking your app’s video and video ads performance and usage. This helps you understand what your users are watching with your app, and how much.
+
+There are several technical advantages to using the native OMSDK over a web player-based solution. We won’t go into these in depth in this document, but here are some of the advantages:
+* Performance
+* Mobile network awareness
+* Frugal memory, thread, and device resource usage
+* Security – webviews / embedded browsers are known weak points
```

Comment:
We don't have them - probably should be outlined.

Reply:
suggest new language.

Reply:
?